types/places.js/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for algolia places 1.16
+// Project: https://github.com/algolia/places
+// Definitions by: Ghizlane LOTFI <https://github.com/ghizlanelotfi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
+
+import {EventEmitter} from 'events';
+
+declare module 'places.js' {
+
+    export default function places(options: IPlacesOptions): EventEmitter;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like in the following:

```ts
namespace places {
    interface PlacesOptions {
        // ...
    }

    export function version(): string;
}

function places(options: PlacesOptions): EventEmitter;
```


[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).